### PR TITLE
used Maven, added serialization, changed dao

### PR DESCRIPTION
**Pull request by Rascass**
I went over to Maven, added serialization/deserialization, changed my hierarchy

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+  <artifactId>HorseClub</artifactId>
+  <version>1.0-SNAPSHOT</version>
+
+  <name>HorseClub</name>
+  <!-- FIXME change it to the project's website -->
+  <url>http://www.example.com</url>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <maven.compiler.source>1.7</maven.compiler.source>
```
Comment (shukal94) [inline pom.xml]: let's use 11

**Review by shukal94 — CHANGES_REQUESTED** [review]
make all of your models serializable
let's define all the services in the separate package
don not see any dao implementation (object or serializable class)

**Comment by Rascass** [comment]
added database connection and changed Mochadao

**Review by shukal94 — APPROVED** [review]

File: src/main/java/com/java/automation/lab/fall/antonyuk/core22/cheker/ReturnTypeValidator.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
+package com.java.automation.lab.fall.antonyuk.core22.cheker;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+public class ReturnTypeValidator {
```
Comment (shukal94) [inline src/main/java/com/java/automation/lab/fall/antonyuk/core22/cheker/ReturnTypeValidator.java:7]: is that date adapter?

**Pull request merged**